packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FGMMarkerUserData.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <GoogleMaps/GoogleMaps.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// Defines user data object for markers.
+@interface FGMMarkerUserData : NSObject
+
+/// The identifier of the marker.
+@property(nonatomic, copy) NSString *markerIdentifier;
+
+/// The identifier of the cluster manager.
+/// This property is set only if the marker is managed by a cluster manager.
+@property(nonatomic, copy, nullable) NSString *clusterManagerIdentifier;
+
+@end
+
+/// Sets MarkerId and optionally ClusterManagerId to GMSMarker UserData.
+///
+/// @param markerIdentifier Identifier of marker.
+/// @param clusterManagerIdentifier Optional identifier of cluster manager.
+/// @param marker GMSMarker object.
```

Comment:
This doesn't tell me anything about its use, only information that is obvious from the function signature to anyone who can read Objecitve-C.